### ISSUE TITLE
Hiera doesn’t do a merge lookup for GPFS bindmounts

### ISSUE THREAD

**Issue by andylytical** [ncsa/puppet-spectrumscale#26]
From Slack:
> qserv-dax01 had node-level Hiera w/ the added /kubernetes mount only, it didn’t get the /datasets mount that was configured at the 02clusters level. I simply added /datasets to the node-level Hiera too and it will enforce both bindmounts

**Comment by andylytical** [comment]
The file `data/common.yaml` has:
```
lookup_options:
    gpfs::(.*)::mountmap:
        merge: hash
```
so it is *supposed* to merge data

**Comment by andylytical** [comment]
Fixed in 32d20ac
